webview-ui/src/components/chat/ChatView.tsx

```diff
@@ -1237,20 +1241,43 @@ const ChatViewComponent: React.ForwardRefRenderFunction<ChatViewRef, ChatViewPro
 					)}
 				</>
 			) : (
-				<div
-					style={{
-						flex: "1 1 0", // flex-grow: 1, flex-shrink: 1, flex-basis: 0
-						minHeight: 0,
-						overflowY: "auto",
-						display: "flex",
-						flexDirection: "column",
-						paddingBottom: "10px",
-					}}>
-					{telemetrySetting === "unset" && <TelemetryBanner />}
-					{showAnnouncement && <Announcement version={version} hideAnnouncement={hideAnnouncement} />}
-
-					<RooHero />
-					{taskHistory.length > 0 && <HistoryPreview showHistoryView={showHistoryView} />}
+				<div className="flex-1 min-h-0 overflow-y-auto flex flex-col gap-4">
+					{/* Moved Task Bar Header Here */}
+					{tasks.length !== 0 && (
+						<div className="flex text-vscode-descriptionForeground w-full mx-auto px-5 pt-3">
+							<div className="flex items-center gap-1 cursor-pointer" onClick={toggleExpanded}>
```

Comment:
What do you think about adding a title tag to this? It felt a little non-obvious seeing it for the first time (in the view where I had more than 10 tasks)

Reply:
It should start in expanded view so people will clue into what it does when they click on it... but I suppose it could be done. Let's punt this one because I want to see how the community reacts and I don't think it's a big one.